server/utils/personUtils.test.ts

```diff
@@ -90,6 +90,146 @@ describe('PersonUtils', () => {
+            {
+              key: { text: 'Conditional release date' },
+              value: { text: '31 October 2024' },
+            },
+            {
+              key: { text: 'Tariff end date' },
+              value: { text: '31 October 2028' },
+            },
+            {
+              key: { text: 'Sentence expiry date' },
+              value: { text: '31 October 2027' },
+            },
+            {
+              key: {
+                html: 'Parole eligibility date<br><span class="govuk-!-font-size-16 govuk-!-font-weight-regular release-dates__earliest-indicator">(Earliest release date)</span>',
+              },
+              value: { text: '31 October 2026' },
+            },
+            {
+              key: { text: 'Home detention curfew eligibility date' },
+              value: { text: '31 October 2025' },
+            },
+          ])
+        })
+      })
+
+      describe('and without a parole eligibility date', () => {
+        it('indicates that parole eligibility date is missing, identifies conditional release date as the earliest release date, and formats release date information in the appropriate format for passing to a GOV.UK Summary List nunjucks macro', () => {
+          const person = { ...personWithAllDatesAndDeterminateSentence, paroleEligibilityDate: undefined }
+
+          expect(PersonUtils.releaseDatesSummaryListRows(person)).toEqual([
+            {
+              key: {
+                html: 'Conditional release date<br><span class="govuk-!-font-size-16 govuk-!-font-weight-regular release-dates__earliest-indicator">(Earliest release date)</span>',
+              },
+              value: { text: '31 October 2024' },
+            },
+            {
+              key: { text: 'Tariff end date' },
+              value: { text: '31 October 2028' },
+            },
+            {
+              key: { text: 'Sentence expiry date' },
+              value: { text: '31 October 2027' },
+            },
+            {
+              key: { text: 'Parole eligibility date' },
+              value: { text: 'There is no record for this date' },
+            },
+            {
+              key: { text: 'Home detention curfew eligibility date' },
+              value: { text: '31 October 2025' },
+            },
+          ])
+        })
+      })
+    })
+
+    describe('when dates are missing', () => {
+      it('indicates that they are missing', () => {
+        const person = {
+          ...personWithAllDatesAndDeterminateSentence,
+          conditionalReleaseDate: undefined,
+          homeDetentionCurfewEligibilityDate: undefined,
+          paroleEligibilityDate: undefined,
+          sentenceExpiryDate: undefined,
+          tariffDate: undefined,
+        }
+
+        expect(PersonUtils.releaseDatesSummaryListRows(person)).toEqual([
+          {
+            key: { text: 'Conditional release date' },
+            value: { text: 'There is no record for this date' },
```

Comment:
I thought we were leaving dates off if we didn't have a record of them. I can see the prototype says otherwise. Just wanted to make sure that was the more recent decision?

Reply:
Yep we discussed that at the design/dev chat on Wednesday